life/original/06-supplement.adoc

```diff
@@ -249,8 +299,11 @@ tender in mercy to all that faithfully serve, worship and obey him.
 
 My spirit salutes thee and dear children, etc.
```

Comment:
I would probably make this a SSCL in both the original and modernized. Other than that, everything looks great. Nice job.